bamboo/common_python/tools.py

```diff
@@ -105,9 +105,9 @@ def get_command(cluster,
 
         # Create run command
         if command_allocate == '':
-            command_run = 'srun'
+            command_run = 'srun --mpibind=off'
```

Comment:
@forsyth2 why does this if/else logic do the same thing? I need to test this mpibind=off on catalyst before we merge this by the way. But we do need it (at least for now) on pascal/surface

Reply:
It doesn't actually do the same thing. The `else` block puts a space in front

Reply:
Because if there's a `salloc` command before, we need a space before `srun`.

Reply:
wow totally missed that subtlety. makes sense.